components/arm/arm.go

```diff
@@ -146,30 +144,20 @@ func NamesFromRobot(r robot.Robot) []string {
 	return robot.NamesByAPI(r, API)
 }
 
-// CreateStatus creates a status from the arm. This will report calculated end effector positions even if the given
-// arm is perceived to be out of bounds.
-func CreateStatus(ctx context.Context, a Arm) (*pb.Status, error) {
-	model := a.ModelFrame()
-	joints, err := a.JointPositions(ctx, nil)
-	if err != nil {
-		return nil, err
-	}
-
-	var endPosition *v1.Pose
-	if endPose, err := referenceframe.ComputeOOBPosition(model, joints); err == nil {
-		endPosition = spatialmath.PoseToProtobuf(endPose)
-	}
-
-	var jointPositions *pb.JointPositions
-	if jp, err := referenceframe.JointPositionsFromInputs(model, joints); err == nil {
-		jointPositions = jp
-	}
+// GoToWaypoints will visit in turn each of the joint position waypoints generated by a motion planner.
+func GoToWaypoints(ctx context.Context, a Arm, waypoints [][]referenceframe.Input) error {
```

Comment:
do we need this?

Reply:
Don't think so, may have been a rebase conflict.